autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1426,12 +1440,10 @@ mod tests {
         client.remove_user(&uaid).await
     }
 
-    /*
-    // XXX: uncomment after the uaid clashing fix
     #[actix_rt::test]
     async fn add_user_existing() {
         let client = new_client().unwrap();
-        let uaid = Uuid::parse_str(TEST_USER).unwrap();
+        let uaid = Uuid::new_v4();
```

Comment:
Not critical, but I still would like to tag these just in case someone does something bad like run unit tests against a production database.